PetAdoptions/cdk/pet_stack/lib/services.ts

```diff
@@ -443,8 +443,8 @@ export class Services extends Stack {
                 connectionType: 'CONNECT_SSM',
                 repositories: [
                     {
-                        repositoryUrl: "https://git-codecommit." + region + ".amazonaws.com/v1/repos/event-source",
-                        pathComponent: "/codecommit/pet_stack"
+                        repositoryUrl: "https://github.com/aws-samples/one-observability-demo.git",
+                        pathComponent: "/one-observability-demo"
```

Comment:
is this where the app will be cloned on cloud9?

Reply:
correct